charts/camunda-platform/templates/ingress.yaml

```diff
@@ -85,7 +85,7 @@ spec:
           {{- if and .Values.connectors.enabled .Values.connectors.contextPath }}
           - backend:
               service:
-                name: {{ include "connectors.fullname" . }}
+                name: {{ template "connectors.fullname" . }}
```

Comment:
Why is that change? :thinking:

Reply:
The change from above: `{{- print "http://" -}}{{- template "connectors.fullname" . -}}:{{- .Values.connectors.service.serverPort -}}` with `include` returns empty value. With `template` though, it does return correct value.

I assumed here is the same thus to be a bug.

Reply:
The result is the same with Helm v3.9.3.
But let's keep it using `template` since it's working as well as the rest of the components using it also.